CHANGELOG.md

```diff
@@ -16,8 +16,9 @@
 - 🔧 reorg individual configs to `conf/modules/` named configs [#459](https://github.com/nf-core/methylseq/pull/469)
 - 🔧 `run_preseq` param + skip preseq/lcextrap module by default [#458](https://github.com/nf-core/methylseq/pull/470)
 - 🔧 `run_qualimap` param + skip qualimap module by default [#367](https://github.com/nf-core/methylseq/pull/471)
-- 🔧 Raised Nextflow version requirement to `24.10.2`
+- 🔧 Raised Nextflow version requirement to `24.10.2` in CI
```

Comment:
This was actually also applied to `manifest` which means that users can only use the pipeline with Nextflow >= 24.10.2

Reply:
I see...cool....I ll update this in the next release PR